README-de.md

```diff
@@ -0,0 +1,83 @@
+# Kubernetes Dokumentation
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+Herzlich willkommen! Dieses Repository enthält alle Assets, die zur Erstellung der [Kubernetes-Website und Dokumentation](https://kubernetes.io/) erforderlich sind. Wir freuen uns sehr, dass Sie dazu beitragen wollen!
+
+## Beiträge zur Dokumentation
+
+Sie können auf die Schaltfläche **Fork** im oberen rechten Bereich des Bildschirms klicken, um eine Kopie dieses Repositorys in Ihrem GitHub-Konto zu erstellen. Diese Kopie wird als *Fork* bezeichnet. Nehmen Sie die gewünschten Änderungen an Ihrem Fork vor. Wenn Sie bereit sind, diese Änderungen an uns zu senden, gehen Sie zu Ihrem Form und erstellen Sie eine neue Pull-Anforderung, um uns darüber zu informieren.
+
+Sobald Ihre Pull-Anfrage erstellt wurde, übernimmt ein Rezensent von Kubernetes die Verantwortung für klares, umsetzbares Feedback. Als Eigentümer des Pull-Request **liegt es in Ihrer Verantwortung Ihren Pull-Reqest enstsprechend des Feedbacks, dass Sie vom Kubernetes-Reviewer erhalten haben abzuändern.** Beachten Sie auch, dass Sie am Ende mehr als einen Rezensenten von Kubernetes erhalten, der Ihnen Feedback gibt, oder dass Sie Rückmeldungen von einem Rezensenten von Kubernetes erhalten, der sich von demjenigen unterscheidet, der ursprünglich für das Feedback zugewiesen wurde. In einigen Fällen kann es vorkommen, dass einer Ihrer Prüfer bei Bedarf eine technische Überprüfung von einem [Kubernetes Techn-Reviewer] (https://github.com/kubernetes/website/wiki/tech-reviewers) anfordert. Reviewer geben ihr Bestes, um zeitnah Feedback zu geben, die Antwortzeiten können jedoch je nach den Umständen variieren.
+
+Weitere Informationen zum Beitrag zur Kubernetes-Dokumentation finden Sie unter:
+
+* [Mitwirkung beginnen](https://kubernetes.io/docs/contribute/start/)
+* [Ihre Dokumentationsänderungen bereitstellen](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Seitenvorlagen verwenden](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Dokumentationsstil-Handbuch](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Übersetzung der Kubernetes-Dokumentation](https://kubernetes.io/docs/contribute/localization/)
+
+## `README.md`'s Localizing Kubernetes Documentation
+
+### Koreanisch
+
+Lesen Sie die Übersetzung von `README.md` und detailliertere Anleitungen für koreanische Mitwirkende auf der Seite [Koreanischen README] (README-ko.md).
+
+Sie können die Betreuer der koreanischen Lokalisierung erreichen unter:
+
+* June Yi ([GitHub - @gochist](https://github.com/gochist))
+* [Slack Kanal](https://kubernetes.slack.com/messages/kubernetes-docs-ko)
+
```

Comment:
Vielleicht auch Deutsche hinzufügen?

Reply:
Thanks for the suggestion. This is definitely a good idea and something which is good to have updated indeed as only the Korean localization is listed right now.

It may be good to update it in a separate PR  (the English `README.md` as well) and just get the initial translation started for now.